projects/styles/markup/tui-island.less

```diff
@@ -175,4 +175,19 @@
         width: ~'calc(100% + 2.375rem);'; // margin compensation
         margin: @padding-large calc(-1.25rem + 1px) calc(-1.25rem + 1px); // 1px is compensations of border
     }
+
+    /* Artificially increased specificity */
+    &__marker&__marker {
```

Comment:
Why do we need this all of a sudden? :)

Reply:
Otherwise, we can't override these lines :(

https://github.com/Tinkoff/taiga-ui/blob/9a7fa2aa170353e68d086952812b99b18a666adc/projects/kit/components/marker-icon/marker-icon.style.less#L17-L18

Reply:
done ✅